dashboard/src/common/SubgraphLogo.tsx

```diff
@@ -4,40 +4,42 @@ const subgraphLogoMap: Record<string, string> = {
   liquity: "https://messari.io/asset-images/d2bc5118-39e8-4b2c-bdea-a7e0c43536f3/128.png?v=2",
-  makerDAO: "https://messari.io/asset-images/4758c080-821e-4a19-bbae-4df59682d229/128.png?v=2",
-  beltFinance: "https://assets.coingecko.com/coins/images/14319/small/belt_logo.jpg?1615387083",
-  stakeDAO: "https://assets.coingecko.com/coins/images/13724/small/stakedao_logo.jpg?1611195011",
+  makerdao: "https://messari.io/asset-images/4758c080-821e-4a19-bbae-4df59682d229/128.png?v=2",
+  "belt-finance": "https://assets.coingecko.com/coins/images/14319/small/belt_logo.jpg?1615387083",
+  "stake-dao": "https://assets.coingecko.com/coins/images/13724/small/stakedao_logo.jpg?1611195011",
   tokemak: "https://messari.io/asset-images/39d6707f-c1f0-4ef9-b916-a3613dc7bd0f/128.png?v=2",
   yearn: "https://messari.io/asset-images/75af0d92-7ec7-4279-bd5b-05eafa1090bf/128.png?v=2",
   curve: "https://assets.coingecko.com/coins/images/12124/small/Curve.png?1597369484",
-  ellipsis: "https://assets.coingecko.com/coins/images/25444/small/ellipsis-light_%281%29.png?1651786591",
+  "ellipsis-finance": "https://assets.coingecko.com/coins/images/25444/small/ellipsis-light_%281%29.png?1651786591",
   rari: "https://assets.coingecko.com/coins/images/12900/small/Rari_Logo_Transparent.png?1613978014",
-  inverse: "https://assets.coingecko.com/coins/images/14205/small/inverse_finance.jpg?1614921871",
+  "inverse-finance": "https://assets.coingecko.com/coins/images/14205/small/inverse_finance.jpg?1614921871",
   convex: "https://assets.coingecko.com/coins/images/15585/small/convex.png?1621256328",
   aurigami: "https://assets.coingecko.com/coins/images/24074/small/EbB5N8IN_400x400.jpg?1646230298",
-  cream: "https://assets.coingecko.com/coins/images/11976/small/Cream.png?1596593418",
-  ironbank: "https://assets.coingecko.com/coins/images/22902/small/ironbank.png?1642872464",
-  traderjoe: "https://assets.coingecko.com/markets/images/692/small/traderjoe.png?1628152581",
+  "cream-finance": "https://assets.coingecko.com/coins/images/11976/small/Cream.png?1596593418",
+  "iron-bank": "https://assets.coingecko.com/coins/images/22902/small/ironbank.png?1642872464",
+  "trader-joe": "https://assets.coingecko.com/markets/images/692/small/traderjoe.png?1628152581",
   sushiswap: "https://assets.coingecko.com/markets/images/576/small/2048x2048_Logo.png?1609208464",
   quickswap: "https://assets.coingecko.com/coins/images/25393/small/quickswap.jpg?1651680141",
   solarbeam: "https://assets.coingecko.com/coins/images/18260/small/solarbeamlogo.png?1636080005",
 };
 
+// used for protocols with multiple versions
 const subgraphMap: Record<string, string> = {
-  aaveV2: "aave",
-  balancerV2: "balancer",
-  BENQI: "benqi",
-  uniswapV3: "uniswap",
-  uniswapV2: "uniswap",
-  yearnV2: "yearn",
+  "aave-v2": "aave",
+  "balancer-v2": "balancer",
+  "uniswap-v3": "uniswap",
+  "uniswap-v2": "uniswap",
+  "rari-vaults-v1": "rari",
```

Comment:
I thought the slug for rari vaults is just `rari-vaults`. Can you confirm?

Reply:
The slug currently used in the "Protocol" entity on rari is `rari-vaults-v1`

Reply:
ah yes, the change hasn't been merged. Nvm